macro/src/span.rs

```diff
@@ -0,0 +1,93 @@
+use proc_macro::Span;
+use std::collections::hash_map::DefaultHasher;
+use std::env;
+use std::hash::{Hash, Hasher};
+
+fn seed() -> impl Hash {
+    env::var_os("CONST_RANDOM_SEED")
```

Comment:
What should we do if this is not set?

Reply:
That's fine, this is a way to plug in optional extra entropy if the build system wants. It'll hash `None` if not set. I think for my use case we would *not* be setting this (but it was suggested in https://github.com/tkaitchuck/constrandom/pull/13#issuecomment-605515816).